src/Explorer/Controls/Settings/SettingsRenderUtils.tsx

```diff
@@ -0,0 +1,372 @@
+  </span>
+);
+
+export const getThroughputApplyShortDelayMessage = (
+  isAutoscale: boolean,
+  throughput: number,
+  throughputUnit: string,
+  databaseName: string,
+  collectionName: string,
+  targetThroughput: number
+): JSX.Element => (
+  <span>
+    A request to increase the throughput is currently in progress. This operation will take some time to complete.
+    <br />
+    Database: {databaseName}, Container: {collectionName}{" "}
+    {getCurrentThroughput(isAutoscale, throughput, throughputUnit, targetThroughput)}
+  </span>
+);
+
+export const getThroughputApplyLongDelayMessage = (
+  isAutoscale: boolean,
+  throughput: number,
+  throughputUnit: string,
+  databaseName: string,
+  collectionName: string,
+  requestedThroughput: number
+): JSX.Element => (
+  <span>
+    A request to increase the throughput is currently in progress. This operation will take 1-3 business days to
+    complete. View the latest status in Notifications.
+    <br />
+    Database: {databaseName}, Container: {collectionName}{" "}
+    {getCurrentThroughput(isAutoscale, throughput, throughputUnit, requestedThroughput)}
+  </span>
+);
+
+export const getToolTipContainer = (content: string | JSX.Element): JSX.Element =>
+  content ? <span>{content}</span> : undefined;
+
+export const conflictResolutionLwwTooltip: JSX.Element = (
+  <span>
+    Gets or sets the name of a integer property in your documents which is used for the Last Write Wins (LWW) based
+    conflict resolution scheme. By default, the system uses the system defined timestamp property, _ts to decide the
+    winner for the conflicting versions of the document. Specify your own integer property if you want to override the
+    default timestamp based conflict resolution.
+  </span>
+);
+
+export const conflictResolutionCustomToolTip: JSX.Element = (
+  <span>
+    Gets or sets the name of a stored procedure (aka merge procedure) for resolving the conflicts. You can write
+    application defined logic to determine the winner of the conflicting versions of a document. The stored procedure
+    will get executed transactionally, exactly once, on the server side. If you do not provide a stored procedure, the
+    conflicts will be populated in the
+    <a className="linkDarkBackground" href="https://aka.ms/dataexplorerconflics" rel="noreferrer" target="_blank">
+      {` conflicts feed`}
+    </a>
+    . You can update/re-register the stored procedure at any time.
+  </span>
+);
+
+export const changeFeedPolicyToolTip: JSX.Element = (
+  <span>
+    Enable change feed log retention policy to retain last 10 minutes of history for items in the container by default.
+    To support this, the request unit (RU) charge for this container will be multiplied by a factor of two for writes.
+    Reads are unaffected.
+  </span>
+);
+
+const dirtyTextFieldColor = "#9b4f96";
+
+export const getTextFieldStyles = (statefulValue?: StatefulValue<unknown>): Partial<ITextFieldStyles> => {
+  return {
```

Comment:
nit: to return the object, you can just do: ` export const blah = () => ({ field1:"foo", field2: "bar" });`